src/docs/product/releases/adding-commits/index.mdx

```diff
@@ -1,60 +1,27 @@
 ---
-title: Suspect Commits
+title: Adding Commits
```

Comment:
We call this page adding commits, but then we only refer to the thing we're doing as associating commits on the rest of the page. I think we should should probably rename the page and update the file name.
```suggestion
title: Associate Commits
```